### ISSUE TITLE
Consider providing NerdFonts compatible releases

### ISSUE THREAD

**Issue by lzybkr** [MonoLisaFont/feedback#57]
Popular tools like [starship](https://github.com/starship/starship) and [lsd](https://github.com/Peltoche/lsd) require glyphs from [NerdFonts](https://www.nerdfonts.com/).

NerdFonts provides patched versions of most popular fonts, but obviously that isn't possible for a paid font. While one can patch MonoLisa themselves, it's a bit of a pain to do so, it would be nice if a patched version was provided with each release.

**Comment by polarmutex** [comment]
second this

**Comment by jjpiv** [comment]
Very much want

**Comment by jjpiv-xerox** [comment]
@lzybkr I can attest to the patching process being a pain as I've not found a way to get this working. The documentation seems fragmented and somewhat incoherent. Do you have any insight to share about how you were able to patch a font?

**Comment by jjpiv** [comment]
@tiagovla Thank you so much for the help! I don't know why it was so difficult the last time I attempted to do this, but your explanation made all the difference.

**Comment by miszo** [comment]
I've prepared NerdFont patch today, and it wasn't that hard. But to be honest, I would be over the moon and forever grateful if the NerdFont glyphs would be available out of the box.

**Comment by jjpiv** [comment]
Yeah, it really isn't that hard. However, not needing to patch the font as new releases are published would be quite nice.

**Comment by bebraw** [comment]
It turns out that for some editors/systems, it's possible to use a fallback font that provides Nerd Fonts symbols. The technique is explained in this [issue comment](https://github.com/MonoLisaFont/feedback/issues/53#issuecomment-1336163548). It's known to work in Kitty terminal and VS Code at least.

**Comment by mikesmithgh** [comment]
> It turns out that for some editors/systems, it's possible to use a fallback font that provides Nerd Fonts symbols. The technique is explained in this [issue comment](https://github.com/MonoLisaFont/feedback/issues/53#issuecomment-1336163548). It's known to work in Kitty terminal and VS Code at least.

Kitty automatically detects Nerd Fonts. Since I have FiraCode Nerd Font installed, all icons fall back to FiraCode.
https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font
>If you are trying to use a font patched with [Nerd Fonts](https://nerdfonts.com/) symbols, don’t do that as patching destroys fonts. There is no need, simply install the standalone Symbols Nerd Font Mono (the file NerdFontsSymbolsOnly.zip from the [Nerd Fonts releases page](https://github.com/ryanoasis/nerd-fonts/releases)). kitty should pick up symbols from it automatically, and you can tell it to do so explicitly in case it doesn’t with the [symbol_map](https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.symbol_map) directive:

**Comment by Lzyct** [comment]
Any update about this?

**Comment by bebraw** [comment]
@Lzyct Yes, see the instructions at https://www.monolisa.dev/faq#how-to-enable-nerd-fonts-with-monolisa .

**Comment by Lzyct** [comment]
> @Lzyct Yes, see the instructions at https://www.monolisa.dev/faq#how-to-enable-nerd-fonts-with-monolisa .

So, we need to build by ourselves.
Every time the font update we need to patch it?

**Comment by bebraw** [comment]
@Lzyct You would have to patch it until we find a better way, yes. Thankfully the external scripts are simple to use.

**Comment by Lzyct** [comment]
> @Lzyct You would have to patch it until we find a better way, yes. Thankfully the external scripts are simple to use.

Okay, can't wait for it. The script is simple but takes time to patch it

**Comment by dylanh724** [comment]
For now, here's a Windows workaround:
```powershell
Windows Registry Editor Version 5.00

; More info here: https://msdn.microsoft.com/en-us/library/windows/desktop/dd317836(v=vs.85).aspx

[HKEY_CURRENT_USER\EUDC\1252]
"SystemDefaultEUDCFont"="FiraCodeNerdFontMono-Regular.ttf"
```

This will universally fallback to a nerd font. In this case, I fallback to FiraCode Mono. Edit above to your liking.

Rename to .reg and run (it'll prompt admin).

For the price, though, I would indeed expect patched fonts for me for each release. Feels like it'd be part of the premium package, watering down that premium feel when you have to go through this trouble.
